auth-web/src/router.ts

```diff
@@ -25,6 +26,7 @@ const routes = [
   { path: '/', component: AuthHome },
   { path: '/home', component: Home },
   { path: '/businessprofile', component: BusinessProfile, meta: { requiresAuth: true } },
+  { path: '/main', component: Template, meta: { requiresAuth: false } },
```

Comment:
It would need auth (requiresAuth true)

Reply:
Yeah left auth as false for now just so we can preview it without signing in for now.  Will def have to be changed to true before moving to prod though.